server/index.js

```diff
@@ -198,6 +197,25 @@ app.post('/api/orders', (req, res, next) => {
   }
 });
 
+app.delete('/api/cart/:cartItemId', (req, res, next) => {
+  const cartItemId = req.params.cartItemId;
+  const deleteSQL = `
+  DELETE FROM "cartItems"
+  WHERE "cartItemId" = $1
+  RETURNING *
+  `;
```

Comment:
You should also include an `AND` so that people can't delete each other's cart items
```sql
delete from "cartItems"
 where "cartItemId" = $1
   and "cartId" = $2
```